x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/index.js

```diff
@@ -170,15 +149,10 @@ export class InnerCustomPlot extends PureComponent {
       return null;
     }
 
-    const staticPlot = (
-      <StaticPlot
-        noHits={noHits}
-        plotValues={plotValues}
-        series={enabledSeries}
-        tickFormatY={this.props.tickFormatY}
-        tickFormatX={this.props.tickFormatX}
-      />
-    );
+    const noHits =
+      flatten(enabledSeries.map(s => s.data)).filter(coord =>
+        isValidCoordinateValue(coord.y)
+      ).length === 0;
```

Comment:
nit: use `some` instead of `filter` to return a boolean (it's faster too):

```js
const noHits = !flatten(enabledSeries.map(s => s.data))
  .some(coord => isValidCoordinateValue(coord.y))
```

or `every` depending on your prefs:
```js
const noHits = flatten(enabledSeries.map(s => s.data))
  .every(coord => !isValidCoordinateValue(coord.y))
```